### PR TITLE
Support for OSD blackbox logging status element

### PR DESCRIPTION
**Pull request by atomgomba**
This PR adds UI to support the changes introduced by related firmware changes:
https://github.com/betaflight/betaflight/pull/6177

### CODE REVIEW

**Comment by McGiverGim** [comment]
The problem with this is that the OSD elements must maintain the same order than in the firmware to work, and we don't have the OSD_ANTI_GRAVITY element added in the configurator. I will try to add it, if not this CAN NOT be merged.

**Comment by McGiverGim** [comment]
Added PR for the ANTI_GRAVITY element. After merged, you will need to rebase your code, and use 330 as `draw_order`.

File: src/js/tabs/osd.js [hunk 1]
```diff
@@ -764,7 +764,15 @@ OSD.constants = {
       preview: function(osd_data) {
         return OSD.generateTemperaturePreview(osd_data, 33);
       }
-    }
+    },
+    LOG_STATUS: {
+      name: 'LOG_STATUS',
+      desc: 'osdDescElementLogStatus',
+      default_position: -1,
+      draw_order: 430,
```
Comment (McGiverGim) [inline src/js/tabs/osd.js]: After the ANTI_GRAVITY PR is merged, this draw_order need to be 330.

**Comment by atomgomba** [comment]
I rebased and force pushed to avoid merging. `draw_order` has also been changed to 330.

**Comment by McGiverGim** [comment]
@atomgomba it seems that there are some conflicts. Could you rebase it again?

**Comment by atomgomba** [comment]
@McGiverGim done

File: src/js/tabs/osd.js [hunk 2]
```diff
@@ -1014,14 +1022,20 @@ OSD.chooseFields = function () {
                   OSD.constants.DISPLAY_FIELDS = OSD.constants.DISPLAY_FIELDS.concat([
                     F.ANTI_GRAVITY
                   ]);
-                  if (semver.gte(CONFIG.apiVersion, "1.40.0")) {
-                    OSD.constants.DISPLAY_FIELDS = OSD.constants.DISPLAY_FIELDS.concat([
-                        F.G_FORCE
-                    ]);
+                  if (semver.gte(CONFIG.apiVersion, "1.39.0")) {
+                      OSD.constants.DISPLAY_FIELDS = OSD.constants.DISPLAY_FIELDS.concat([
+                        F.ANTI_GRAVITY
```
Comment (McGiverGim) [inline src/js/tabs/osd.js]: Something went wrong in te rebase, the version 1.39 with ANTI_GRAVITY element appears twice.

Comment (atomgomba) [reply src/js/tabs/osd.js]: Okay, I removed the duplicate and changed the order of the elements to match the ordering at other places.

**Comment by McGiverGim** [comment]
@mikeller we were too fast merging https://github.com/betaflight/betaflight-configurator/pull/1099 before the firmware PR. So caution when merging all this PR, to not need to change this PR again we need to merge first the https://github.com/betaflight/betaflight/pull/6233 and then the https://github.com/betaflight/betaflight/pull/6177 fixing any conflict...

**Comment by mikeller** [comment]
Erg, yes, now I see. Adding the 'Needs coordination with betaflight' label makes it easier to spot these dependencies.

**Pull request merged**